airbyte-webapp/src/views/Connector/ServiceForm/serviceFormContext.tsx

```diff
@@ -83,18 +85,31 @@ export const ServiceFormContextProvider: React.FC<ServiceFormContextProviderProp
 
   const authFieldsToHide = useMemo(
     () =>
-      Object.values(serverProvidedOauthPaths(selectedConnector)).map((f) =>
-        makeConnectionConfigurationPath(f.path_in_connector_config)
-      ),
-    [selectedConnector]
+      isAuthFlowSelected
+        ? Object.values(serverProvidedOauthPaths(selectedConnector)).map((f) =>
+            makeConnectionConfigurationPath(f.path_in_connector_config)
+          )
+        : [],
+    [selectedConnector, isAuthFlowSelected]
   );
 
+  const authErrors = useMemo(() => {
+    //key of field path, value of error code
```

Comment:
I've just merged https://github.com/airbytehq/airbyte/pull/15968 which will fail this comment (due to no space) , sorry :D Could you maybe rebase with master and have this (auto)fixed so this won't break master if we'll merge it.